mock/blockContainerStub.go

```diff
@@ -0,0 +1,19 @@
+package mock
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+)
+
+type BlockContainerStub struct {
```

Comment:
Dummy comment here. Also, could've made this directly a mock, since you use it this way

Reply:
done